Resources/Prototypes/_Mono/Entities/Objects/Weapons/Melee/sword.yml

```
@@ -178,8 +178,106 @@
       types:
         Slash: 35
         Structural: 40
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
   - type: Sharp
   - type: DisarmMalus
     malus: 0.6
   - type: Execution
     doAfterDuration: 4.0
+
+# Ullman Blade
+
+- type: entity
+  parent: BaseMeleeWeaponEnergy
+  id: UllmanPlasmaBlade
+  name: U.I. unstable plasma blade
+  description: A unique weapon, using welding fuel as a catalyst to ignite. The unstable plasma bursts forth with a wild, powerful flame to temporarily boost damage output.
+  components:
+  - type: Sprite
+    sprite: _Mono/Objects/Weapons/Melee/ullmansword.rsi
+    layers:
+    - state: e_sword
+    - state: e_sword_blade
+      visible: false
+      shader: unshaded
+      map: [ "blade" ]
+  - type: Clothing
+    sprite: _Mono/Objects/Weapons/Melee/ullmansword.rsi
+    slots:
+    - Back
+    - SuitStorage
+  - type: Item
+    sprite: _Mono/Objects/Weapons/Melee/ullmansword.rsi
+    size: Large
+  - type: MeleeWeapon
+    heavyStaminaCost: 1
+    wideAnimationRotation: -135
+    attackRate: 1.25
+    damage:
+      types:
+        Slash: 30
+        Structural: 75
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
+  - type: ItemToggleMeleeWeapon
+    activatedSoundOnHit:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/eblade1.ogg
+      params:
+        variation: 0.250
+    activatedSoundOnHitNoDamage:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/eblade1.ogg
+      params:
+        variation: 0.250
+        volume: -10
+    activatedSoundOnSwing:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/swing.ogg
+      params:
+        variation: 0.125
+    activatedDamage:
+        types:
+            Slash: 30
+            Heat: 50
+            Structural: 300
+  - type: ItemToggle
+    predictable: false
+    soundActivate:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/on.ogg
+    soundDeactivate:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/off.ogg
+  - type: ItemToggleActiveSound
+    activeSound:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/hum.ogg
+  - type: SurgeryTool # Shitmed
+    startSound:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/hum.ogg
+    endSound:
+      path: /Audio/_Goobstation/Weapons/EswordWelder/eblade1.ogg
+      params:
+        variation: 0.250
+  - type: WeaponMeleeCharge
+    activeTime: 2
+    cooldown: 5
+    cooldownDamagePenalty:
+      types:
+        Slash: -10
+  - type: ItemToggleHot
+  - type: ComponentToggler
+    components:
+    - type: Reflect
+      reflectProb: 0.5
+      reflects:
+        - Energy
+  - type: EnergySword
+    activatedColor: LightBlue
+    colorOptions: []
+  - type: Tag # WD EDIT
+    tags:
+    - EnergySword
+  - type: Sharp
+  - type: DisarmMalus
+    malus: 0.6
+  - type: Execution
+    doAfterDuration: 4.0
+  - type: UseDelay
+    delay: 5
```

make it 1 since `WeaponMeleeCharge` already handles thing.
```suggestion
  - type: UseDelay
    delay: 1
```